packages/ui5-library-writer/templates/common/ui5.yaml

```diff
@@ -0,0 +1,18 @@
+specVersion: '2.2'
+metadata:
+  name: "<%= libraryName %>"
+type: library
+framework:
+  name: <%= framework %>
+  version: <%= frameworkVersion %>
+  libraries:
+    - name: sap.ui.core
+    - name: themelib_sap_belize
+    - name: themelib_sap_fiori_3
```

Comment:
normally we put this in the ui5-local.yaml
and in the ui5.yaml we load from cdn via the proxy middleware

Reply:
```
server:
  customMiddleware:
    - name: fiori-tools-proxy
      afterMiddleware: compression
      configuration:
        ignoreCertError: false # If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted
        ui5:
          path:
            - /resources
            - /test-resources
          url: https://ui5.sap.com
```

Reply:
Including this caused errors I could not resolve for now

Reply:
OK. As discussed fine to leave it like this.

Reply:
The fix for that one was just merged to `main` today, as soon as the `@sap/ux-ui5-tooling` is consuming the latest version of the `@sap-ux/ui5-proxy-middleware` you should be able to change it like @devinea suggested.

@cianmSAP could you add this as additional task to #1006 and fix it once it is done (so that we don't forget about it once this PR Is merged)